spec/controllers/v1/post911_gi_bill_statuses_controller_spec.rb

```diff
@@ -29,6 +30,16 @@
       end
 
       expect(response).to have_http_status(:ok)
+      response_body = JSON.parse(response.body)
+
+      # assertions that the data returned will not be empty strings
+      expect(response_body['first_name']).not_to be_empty
+      expect(response_body['last_name']).not_to be_empty
+      expect(response_body['date_of_birth']).not_to be_empty
+      expect(response_body['delimiting_date']).not_to be_empty
+      expect(response_body['eligibility_date']).not_to be_empty
+      expect(response_body['enrollments'][0]['begin_date']).not_to be_empty
+      expect(response_body['enrollments'][0]['end_date']).not_to be_empty
```

Comment:
These are the fields which come back from Lighthouse as `dateTimeOfBirth`, `delimitingDateTime`, etc.